package.json

```diff
@@ -8,8 +8,8 @@
     "test": "test"
   },
   "scripts": {
-    "test": "mocha test/* --reporter nyan",
-    "start": "mocha test/* --reporter nyan --watch"
+    "test": "mocha tests --recursive --reporter nyan",
```

Comment:
Grabbed it from the plugin config that would be generated, if you used Yeoman to create the initial plugin:

https://github.com/eslint/generator-eslint/blob/43e6fe26cca2f19628bb94e34e4430710bad9918/plugin/templates/_package.json#L13